src/routes/project/[projectId]/dashboard/+page.ts

```diff
@@ -1,38 +1,47 @@
 import { backendService } from "$lib/grpc-api";
 import type { PageLoad } from "./$types";
-import { type Project, type Project_Paper } from "$lib/model/api/project";
+import {
+    PaperDecision,
+    type Project,
+    type Project_Information_DecisionStatistics,
+} from "$lib/model/api/project";
 import type {
     PaperListEntryInterface,
     ProjectInformationInterface,
     StageProgressInterface,
 } from "$lib/model/component-interfaces";
-import { getStatusText, groupBy } from "$lib/utils/common-helper";
 
-async function groupProjectPapersInStageByDecision(
+async function requestProjectInformation(
     project: Project,
-): Promise<{ [key: string]: Project_Paper[] }> {
-    const papersOfStage = await backendService
-        .getAllProjectPapersForProject({ id: project.id })
-        .response.then((papers) =>
-            papers.projectPapers.filter(
-                (projectPaper) => projectPaper.stage === project.currentStage,
-            ),
-        );
-
-    return groupBy(papersOfStage, getStatusText);
-}
-
-async function requestProjectInformation(project: Project): Promise<ProjectInformationInterface> {
-    // TODO: add a real call to the backend (extend API + extend mock backend), waiting for #7 in snowballr-api
-    const startDate = new Date();
-    startDate.setDate(startDate.getDate() - 5);
-    const daysInStage = 2;
+    decisionStatistics: Project_Information_DecisionStatistics,
+): Promise<ProjectInformationInterface> {
+    const projectInformation = await backendService.getProjectInformation({ projectId: project.id })
+        .response;
+    if (!projectInformation.creationDate) {
+        throw new Error("Couldn't load start date of project.", {
+            cause: "NoDate",
+        });
+    }
+    const startDate = new Date(Number(projectInformation.creationDate.seconds) * 1000);
+    if (!projectInformation.lastStageStarted) {
+        throw new Error("Couldn't load start date of latest stage.", {
+            cause: "NoDate",
+        });
+    }
+    const startDateForStage = new Date(Number(projectInformation.lastStageStarted.seconds) * 1000);
+    const daysInStage = Math.round(
+        Math.abs(new Date().getTime() - startDateForStage.getTime()) / 8.64e7,
```

Comment:
```suggestion
        Math.abs(Date.now() - startDateForStage.getTime()) / 8.64e7,
```
Additionally, please consider using a constant for that magic number.

Reply:
Good idea, I changed it.